lib/blocktree/blocktree.go

```diff
@@ -418,3 +418,10 @@ func (bt *BlockTree) GetBlockRuntime(hash common.Hash) (runtime.Instance, error)
 	}
 	return ins, nil
 }
+
+// ClearRuntimes deletes all runtime instances from BlockTree runtime map
+func (bt *BlockTree) ClearRuntimes() {
+	for key := range bt.runtimes.mapping {
+		delete(bt.runtimes.mapping, key)
+	}
```

Comment:
This is not thread safe, you should instead add a method to `bt.runtimes` locking a mutex and re-making the map (since maps don't shrink when we delete elements)